PixiEditor/Views/ColorPanelController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AvalonDock.Layout;
+using System.Windows;
+
+namespace PixiEditor.Views
+{
+    internal class ColorPanelController
+    {
+        private LayoutAnchorable colorPickerPanel;
+        private LayoutAnchorable colorSlidersPanel;
+        private LayoutAnchorable smallColorPickerPanel;
+
+        public ColorPanelController(LayoutAnchorable colorPickerPanel, LayoutAnchorable colorSlidersPanel, LayoutAnchorable smallColorPickerPanel)
+        {
+            this.colorPickerPanel = colorPickerPanel;
+            this.colorSlidersPanel = colorSlidersPanel;
+            this.smallColorPickerPanel = smallColorPickerPanel;
+        }
+
+        public void DeterminePanelsToDisplay()
+        {
+            if (SystemParameters.PrimaryScreenHeight < 1010)
```

Comment:
I wouldn't use `PrimaryScreenHeight`, I'd expect panels to change depending on the window height